src/hlstrafe.cpp

```diff
@@ -2031,10 +2043,13 @@ namespace HLStrafe
 		if (vars.Frametime != 0.f)
 			curState.PredictThis = State0ms::NOTHING;
 
-		bool reduceWishspeed = playerCopy.Ducking;
+		bool reduceWishspeed = playerCopy.Ducking || (vars.HasStamina && playerCopy.InDuckAnimation);
```

Comment:
What do you think the implication of reducing wishspeed when InDuckAnimation? I am having hard time wrapping around it as I have been comparing the physics but don't know for sure why this is included (in the original cs branch). Maybe I am missing something but removal of this does desync one made with it. In prediction, this slows down 1-2 unit than without, so there is some funky friction stuff.